### PR TITLE
Standardize ticks fontsize

### PR DESCRIPTION
**Pull request by mepland**
Standardize `ticks_fontsize` usage

### CODE REVIEW

File: dtreeviz/trees.py [hunk 1]
```diff
@@ -1526,7 +1527,7 @@ def _ctreeviz_univar(shadow_tree,
             ax.plot([split, split], [*ax.get_ylim()], '--', color=colors['split_line'], linewidth=1)
 
 
-def _ctreeviz_bivar(shadow_tree, fontsize=10, fontname="Arial", show={'title', 'legend', 'splits'},
+def _ctreeviz_bivar(shadow_tree, fontsize=10, ticks_fontsize=10, fontname="Arial", show={'title', 'legend', 'splits'},
```
Comment (parrt) [inline dtreeviz/trees.py:1530]: Should this be ticks_fontsize=8 to be consistent with the ctree_feature_space arg?

Comment (mepland) [reply dtreeviz/trees.py]: Sure, done!

**Comment by parrt** [comment]
I will make the analogous changes actually and close this. I just pushed a fix to remove the fall parameters from our internal implementation methods, but I will just add the tick Font size argument. thanks.

**Comment by parrt** [comment]
@mepland please see https://github.com/parrt/dtreeviz/commit/5cf1b61f2519617435a7a971601c8987789440aa

**Pull request closed without merging**